src/platform_impl/windows/window_state.rs

```diff
@@ -123,6 +122,53 @@ impl WindowState {
         this.window_flags.set(WindowFlags::MARKER_FULLSCREEN, is_fullscreen);
         let new_flags = this.window_flags;
 
+        // Change video mode if necessary
+        if old_flags & WindowFlags::MARKER_FULLSCREEN != new_flags & WindowFlags::MARKER_FULLSCREEN
```

Comment:
Right now, this won't update the video mode if the video mode is changed while fullscreen is active.